src/components/DashboardHome/Sidebar.jsx

```diff
@@ -2,10 +2,27 @@ import React from 'react';
 
 import Links from '../Links/Links';
 import linksData from './sampleLinks';
+import SidebarBlock from './SidebarBlock';
 
 const Sidebar = () => (
   <>
-    <Links id={linksData.id} title={linksData.title} links={linksData.links} />
+    <SidebarBlock title="Program Document" sectionClassName="links">
```

Comment:
"Program Document" should be plural (i.e., "Documents")

Reply:
We're not actually using the `.links` class anywhere in our styles, etc. so we can actually remove it from here. Related, though, we should add some margin between the blocks by adding a class name of `.mb-4` to the first 2 `SidebarBlock`s.